config/locales/en.yml

```diff
@@ -1061,6 +1061,7 @@ en:
       edit:
         order_sure_want_to: Are you sure you want to %{event} this order?
         voucher_tax_included_in_price: "%{label} (tax included in voucher)"
+        additional_tax_included_in_price: "Tax on fees"
```

Comment:
Could you rename this ? it's confusing because `additional` in the context of an adjustment means "not included" in the price. Maybe something simple like `tax_on_fees` ?

Reply:
ok, renamed